fourkitchens/fire
# FIre phpCS command

- Provides new PHPCS command .
This command configures and also test the PHPCS for your drupal project .

## How to test:

-  install this package version into a drupal Project that doesn't have any phpCS configure : `composer require fourkitchens/fire:dev-feature/phpcs-command --dev`
- Once installed ,  execute the command `fire`, into the available commands you should be able to spot this one `local:lint:php`
- now run `fire local:lint:php` or fire `fire phpcs`
- if it is your first time running it , it should ask you some question to create some configuration for you + the phpcs test
- if it is your second time running it , it should only run the phpcs test
